### ISSUE TITLE
Exception in thread "JNativeHook Hook Thread" java.lang.UnsatisfiedLinkError

### ISSUE THREAD

**Issue by dons20** [kwhat/jnativehook#97]
I'm using eclipse to create a project using the jnativehook files, however this error keeps coming up and I'm not sure what to do to fix it. I've tried editing the build path, changing the references and a couple other things. I assume the issue is related to some linking error somewhere in the project but I'm unable to find out why.

Any ideas as to what could be causing the issue? I'm currently trying all the methods I can think of but I hope it isn't a bug. 

`Exception in thread "JNativeHook Hook Thread" java.lang.UnsatisfiedLinkError: org.jnativehook.GlobalScreen$NativeHookThread.enable()v`
`at org.jnativehook.GlobalScreen$NativeHookThread.enable(Native Method)`
`at org.jnativehook.GlobalScreen$NativeHookThread.run(GlobalScreen.java:237)`

**Comment by kwhat** [comment]
Are you using the binary jar or project source?  If you are using the binary jar, you may need to extract the native libraries and set your `java.library.path` property in eclipse.  If you are trying to use the source, you must compile the library for your development library using ant.

**Comment by dons20** [comment]
I will try each method and share my results.

**Comment by dons20** [comment]
Ok after much fiddling with various settings, I finally found a method that actually worked. What I did was, to extract the two folders from the binary jar, then I put them both in a folder inside my project directory (a folder I called `java`). I added the `java` folder to my java build path (Project -> Properties -> Libraries) and I clicked on Add Class Folder. I clicked on the `java` folder, applied it and then I exported the jar file which proceeded to work and I didn't encounter the issue in my tests. 

I did restart Eclipse twice though because it bugged out with finding some package names. But combined with that method, cleaning the project and sorting out all the issues, it worked.
